src/modules/public/customization/sagas/customizationFetchSaga.ts

```diff
@@ -1,21 +1,24 @@
 import { call, put } from 'redux-saga/effects';
+import { sendError } from '../../../';
 import { API, RequestOptions } from '../../../../api';
-import { alertPush } from '../../alert';
-import {
-    customizationData,
-    customizationError,
-} from '../actions';
+import { customizationData, customizationError, CustomizationFetch } from '../actions';
 
 const customizationOptions: RequestOptions = {
     apiVersion: 'applogic',
 };
 
-export function* customizationFetchSaga() {
+export function* customizationFetchSaga(action: CustomizationFetch) {
     try {
         const customization = yield call(API.get(customizationOptions), '/customization');
         yield put(customizationData(customization));
     } catch (error) {
-        yield put(customizationError(error));
-        yield put(alertPush({message: error.message, code: error.code, type: 'error'}));
+        yield put(sendError({
+            error,
+            processingType: 'alert',
+            extraOptions: {
+                type: action.type,
```

Comment:
`type` property should be optional, you don't have to pass it on each `sendError` call.
`type` property should be passed only when it's handled inside `sendError`

Reply:
Do it for all methods

Reply:
Done